Making project for bootcamp on Python

We import pandas and numpy.

In [1]:
import pandas as pd
import numpy as np

It prints type of data.

In [14]:
data = pd.read_csv('/content/train (1).csv')
print("Data is of typr:",type(data))

Data is of typr: <class 'pandas.core.frame.DataFrame'>


Here, code prints the size od data i.e, number of rows and column

In [15]:
print("Data has shape",data.shape)

Data has shape (891, 12)


We see above that the csv file wh have uploaded has 891 rows and 12 columns.

Now we print whole data.

In [16]:
data

PassengerId  Survived  Pclass  ...     Fare Cabin  Embarked
0              1         0       3  ...   7.2500   NaN         S
1              2         1       1  ...  71.2833   C85         C
2              3         1       3  ...   7.9250   NaN         S
3              4         1       1  ...  53.1000  C123         S
4              5         0       3  ...   8.0500   NaN         S
..           ...       ...     ...  ...      ...   ...       ...
886          887         0       2  ...  13.0000   NaN         S
887          888         1       1  ...  30.0000   B42         S
888          889         0       3  ...  23.4500   NaN         S
889          890         1       1  ...  30.0000  C148         C
890          891         0       3  ...   7.7500   NaN         Q

[891 rows x 12 columns]

Now we print first few data from top instead of whole data.

In [17]:
data.head(5)

PassengerId  Survived  Pclass  ...     Fare Cabin  Embarked
0            1         0       3  ...   7.2500   NaN         S
1            2         1       1  ...  71.2833   C85         C
2            3         1       3  ...   7.9250   NaN         S
3            4         1       1  ...  53.1000  C123         S
4            5         0       3  ...   8.0500   NaN         S

[5 rows x 12 columns]

Now we print few data from last rows.

In [7]:
data.tail(6)

PassengerId  Survived  Pclass  ...    Fare Cabin  Embarked
885          886         0       3  ...  29.125   NaN         Q
886          887         0       2  ...  13.000   NaN         S
887          888         1       1  ...  30.000   B42         S
888          889         0       3  ...  23.450   NaN         S
889          890         1       1  ...  30.000  C148         C
890          891         0       3  ...   7.750   NaN         Q

[6 rows x 12 columns]

Herwe we check that we have any null value or not by using "isnull()" and "any()" is use to chech that any column have null or not.

In [11]:
data.isnull().any()

PassengerId    False
Survived       False
Pclass         False
Name           False
Sex            False
Age             True
SibSp          False
Parch          False
Ticket         False
Fare           False
Cabin           True
Embarked        True
dtype: bool

Here we count missing data in each column.

In [13]:
data.isna().sum()

PassengerId      0
Survived         0
Pclass           0
Name             0
Sex              0
Age            177
SibSp            0
Parch            0
Ticket           0
Fare             0
Cabin          687
Embarked         2
dtype: int64